app/Http/Controllers/Api/V1/Integration/SmithsonianIOAAPIClientController.php

```diff
@@ -59,9 +65,10 @@ public function getContentList(Request $request)
     }
 
     /**
+     * Get Smithsonian Content
```

Comment:
Add new line between title, description param and return.

Reply:
Done!